File: Modul-3/Soal 4/PRAK304-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>

int main(){
    int angka;
    printf("Masukkan Sebuah Angka: ");
    scanf("%d", &angka);

    if      (angka == 0){printf("Nol");}
    else if (angka < 0 || angka >= 100){printf("Anda Menginput Melebihi Bilangan");}
    else if (angka <= 9){printf("Satuan");}
    else if (angka == 10){printf("Puluhan");}
    else if (angka <= 19){printf("Belasan");}
    else if (angka <= 99){printf("Puluhan");}
}

File: Modul-3/Soal 4/PRAK304-2410817110009-MuhammadAlfiGunawan.py
nilai = int(input("Masukkan Sebuah Angka: "))

if nilai == 0:
    print("Nol")
elif nilai < 0 or nilai > 99:
    print("Anda Menginput Melebihi Bilangan")
elif nilai <= 9:
    print("Satuan")
elif nilai == 10:
    print("Puluhan")
elif nilai <= 19:
    print("Belasan")
elif nilai <= 99:
    print("Puluhan")